### PR TITLE
bump major for new development work

### PR DESCRIPTION
**Pull request by zburke**

### CODE REVIEW

**Review by MikeTaylor — COMMENTED** [review]
I don't get it. What backwards-incompatible change is this major-version bump signalling?

**Comment by zburke** [comment]
The major version bump happens in a separate commit from the breaking change itself so we can revert things in isolation. e.g. we'll bump react to v18, react-intl to v6, and possibly make some other breaking change as well (like dropping deprecated components or props). 

If we made the version bump as part of the first commit, then made the other commits, then decided to revert the react v18 change, it'd be a mess: we'd have to add a new major-version commit but in the history it would come after making the actual breaking changes. 

TLDR: separate commits are just insurance to help prevent me from shooting myself in the foot (again).

**Comment by MikeTaylor** [comment]
I see. That all makes sense.

(It would have been helpful, though, if the commit comment said "bump major in anticipation of X".)
